task/request.go

```diff
@@ -16,4 +16,10 @@ type Request struct {
 	// Secrets provides the names and values of secrets
 	// that are available to the task execution.
 	Secrets map[string]string `json:"-"`
+
+	// Account provides the account identifier.
+	Account string `json:"account"`
+
+	// Server will assign a request id to track status of the request
+	RequestId string `json:"id"`
```

Comment:
We can change this from `Request.RequestID` to `Request.ID` to better align with the Go naming conventions.

Reply:
Also just a minor note that comments in Go generally start with the name of the field / struct / function per convention. https://go.dev/doc/comment

Example from the Go standard library:

```
// A Request represents an HTTP request ...
type Request struct {
	// Method specifies the HTTP method ...
	Method string

	// URL specifies either ...
	URL *url.URL

	// Header contains ...
	Header Header

	// Body is the request's body.
	Body io.ReadCloser

	// GetBody defines ....
	GetBody func() (io.ReadCloser, error)

	// ContentLength records the ....
	ContentLength int64

	// TransferEncoding lists the ....
	TransferEncoding []string

	// Close indicates whether ....
	Close bool

	// Form contains ...
	Form url.Values

	// PostForm contains ...
	PostForm url.Values

	// MultipartForm is the parsed ...
	MultipartForm *multipart.Form
}
```

So in this case we might do something like this:

```
// ID provides a unique identifier to track the status of the request.
ID string `json:"id"`
```